src/science/roughness/cable_roughness.F90

```diff
@@ -22,17 +23,22 @@
 
 MODULE cable_roughness_module
 
-USE cable_phys_constants_mod, ONLY : CCSD   => CSD 
-USE cable_phys_constants_mod, ONLY : CCRD   => CRD 
-USE cable_phys_constants_mod, ONLY : CCCD   => CCD 
-USE cable_phys_constants_mod, ONLY : CCCW_C => CCW_C 
-USE cable_phys_constants_mod, ONLY : CUSUHM  => USUHM 
-USE cable_phys_constants_mod, ONLY : CVONK   => VONK
-USE cable_phys_constants_mod, ONLY : CA33    => A33 
-USE cable_phys_constants_mod, ONLY : CCTL    =>  CTL  
-USE cable_phys_constants_mod, ONLY : CZDLIN  => ZDLIN
-USE cable_phys_constants_mod, ONLY : CCSW    => CSW
-USE cable_phys_constants_mod, ONLY : CGRAV   => GRAV
+#ifdef UM_CBL
+USE cable_surface_types_mod, ONLY: ICE_SurfaceType => ICE_cable
+#else
+USE grid_constants_mod_cbl,  ONLY: ICE_SurfaceType => ICE_cable
+#endif
```

Comment:
Review. This does not make any sense with what is above in cable_surface_types.F90 and grid_constants_cbl.F90.

It also appears in other PR. Might need revision after the previous PR is in. The other PR will add these to main so these differences are likely to disappear from this PR. We will need to remember to modify these lines in the current branch once main is updated from the other PR #404 .